docs/content/4.sdk/1.index.md

```diff
@@ -8,7 +8,7 @@ The Vue Storefront SDK simplifies the development process by providing pre-built
 
 By utilizing the Vue Storefront SDK, you can establish a type-safe contract between your storefront and various APIs with the help of the Server Middleware acting as a proxy.
 
-:card{to="$base/getting-started" title="Get Started" description="Learn how to initialize the SDK in your storefronts" icon="ic:baseline-rocket"}
+:card{to="/sdk/getting-started" title="Get Started" description="Learn how to initialize the SDK in your storefronts" icon="ic:baseline-rocket"}
```

Comment:
Is base no longer necessary?

Reply:
These docs got moved from the `sdk` repo into here so we need to replace `$base/` with `/sdk`